functions/system.bash

```diff
@@ -314,20 +315,20 @@ Finally, all common serial ports can be made accessible to the openHAB java virt
   fi
 
   if [[ $selection == *"2"* ]]; then
-    if is_pithree || is_pithreeplus; then
+    if is_pithree || is_pithreeplus || is_pifour; then
       #cond_redirect systemctl stop hciuart &>/dev/null
       #cond_redirect systemctl disable hciuart &>/dev/null
-      cond_echo "Adding 'dtoverlay=pi3-miniuart-bt' to /boot/config.txt (RPi3)"
-      if ! grep -q "dtoverlay=pi3-miniuart-bt" /boot/config.txt; then
-        echo "dtoverlay=pi3-miniuart-bt" >> /boot/config.txt
+      cond_echo "Adding 'dtoverlay=miniuart-bt' to /boot/config.txt (RPi3/4)"
+      if ! grep -q "dtoverlay=miniuart-bt" /boot/config.txt; then
+        echo "dtoverlay=miniuart-bt" >> /boot/config.txt
       fi
     else
-      cond_echo "Option only available for Raspberry Pi 3."
+      cond_echo "Option only available for Raspberry Pi 3/4."
     fi
   else
-    if is_pithree || is_pithreeplus; then
-      cond_echo "Removing 'dtoverlay=pi3-miniuart-bt' from /boot/config.txt"
-      sed -i '/dtoverlay=pi3-miniuart-bt/d' /boot/config.txt
+    if is_pithree || is_pithreeplus || is_pifour; then
+      cond_echo "Removing 'dtoverlay=miniuart-bt' from /boot/config.txt"
+      sed -i '/dtoverlay=miniuart-bt/d' /boot/config.txt
```

Comment:
add `^` or something like  `^[:space:]*` to the sed command to ensure comments stay in

Reply:
done